backend/tests/test_api.py

```
@@ -785,5 +785,308 @@ def test_search_with_very_long_query(self, mock_embeddings, mock_summarize,
+    @patch('backend.settings.get_embedding_config_from_ini')
+    def test_get_embedding_settings(self, mock_get_config):
+        """Test getting embedding settings."""
+        mock_get_config.return_value = {
+            'provider_type': 'local',
+            'model_name': 'test-model',
+            'api_key_set': False
+        }
+
+        response = self.client.get("/api/settings/embeddings")
+
+        self.assertEqual(response.status_code, 200)
```

Ruff (0.15.5)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)